media-sound/teamspeak-client-bin/metadata.xml

```diff
@@ -5,14 +5,6 @@
 		<email>ck+gentoo@bl4ckb0x.de</email>
 		<name>Conrad Kostecki</name>
 	</maintainer>
-	<maintainer type="person">
```

Comment:
Why are you removing other maintainers here? I don't see any paper trail for this.

Reply:
Because they didn't reply over a year to the ticket and I wrote them also directly by mail and never got a response. I am sorry, was this wrong in this case?

Reply:
Fixed.